[2021-10-02 19:28:14,550] {taskinstance.py:903} INFO - Dependencies all met for <TaskInstance: my_dag.Training_Model_B 2021-10-02T19:28:13.149551+00:00 [queued]>
[2021-10-02 19:28:14,581] {taskinstance.py:903} INFO - Dependencies all met for <TaskInstance: my_dag.Training_Model_B 2021-10-02T19:28:13.149551+00:00 [queued]>
[2021-10-02 19:28:14,582] {taskinstance.py:1095} INFO - 
--------------------------------------------------------------------------------
[2021-10-02 19:28:14,583] {taskinstance.py:1096} INFO - Starting attempt 1 of 1
[2021-10-02 19:28:14,585] {taskinstance.py:1097} INFO - 
--------------------------------------------------------------------------------
[2021-10-02 19:28:14,598] {taskinstance.py:1115} INFO - Executing <Task(PythonOperator): Training_Model_B> on 2021-10-02T19:28:13.149551+00:00
[2021-10-02 19:28:14,604] {standard_task_runner.py:52} INFO - Started process 20802 to run task
[2021-10-02 19:28:14,615] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'my_dag', 'Training_Model_B', '2021-10-02T19:28:13.149551+00:00', '--job-id', '38', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/my_dag.py', '--cfg-path', '/tmp/tmpp0adg0n2', '--error-file', '/tmp/tmpiadqrr12']
[2021-10-02 19:28:14,617] {standard_task_runner.py:77} INFO - Job 38: Subtask Training_Model_B
[2021-10-02 19:28:14,685] {logging_mixin.py:109} INFO - Running <TaskInstance: my_dag.Training_Model_B 2021-10-02T19:28:13.149551+00:00 [running]> on host d6b4131456ad
[2021-10-02 19:28:14,759] {taskinstance.py:1254} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=my_dag
AIRFLOW_CTX_TASK_ID=Training_Model_B
AIRFLOW_CTX_EXECUTION_DATE=2021-10-02T19:28:13.149551+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-10-02T19:28:13.149551+00:00
[2021-10-02 19:28:14,783] {python.py:151} INFO - Done. Returned value was: None
[2021-10-02 19:28:14,810] {taskinstance.py:1219} INFO - Marking task as SUCCESS. dag_id=my_dag, task_id=Training_Model_B, execution_date=20211002T192813, start_date=20211002T192814, end_date=20211002T192814
[2021-10-02 19:28:14,872] {local_task_job.py:151} INFO - Task exited with return code 0
[2021-10-02 19:28:14,924] {local_task_job.py:261} INFO - 0 downstream tasks scheduled from follow-on schedule check
